src/go/k8s/README.md

```diff
@@ -0,0 +1,53 @@
+# Redpanda operator
+
+[Redpanda operator](https://github.com/vectorizedio/redpanda/src/go/k8s) is Kubernetes Operator
+for managing and automating tasks related to managing Redpanda clusters.
+
+[Redpanda](https://github.com/vectorizedio/redpanda) is a streaming platform for mission critical
+workloads. Kafka® compatible, No Zookeeper®, no JVM, and no code changes required.
+Use all your favorite open source tooling - 10x faster.
+
+## Getting started
+
+### Requirements
+
+* Kubernetes 1.16 or newer
+* kubectl 1.16 or newer
+* kustomize v3.8.7 or newer
+* kind v0.9.0 or newer
+
+### Installation
+
+#### Local installation
+
+First clone the repo
+
+```
+git clone https://github.com/vectorizedio/redpanda.git
+```
+
+Create local Kubernetes cluster using KIND
+
+```
+export KUBECONFIG=your/path/to/kubeconfig.yaml
+kind create cluster --config kind.yaml
+```
+
+You can simply deploy the latest version stored in GCR
```

Comment:
oh it is? We need a public image then at one point as well. If it's private we should probably not even mention it?

Reply:
Ok I will remove this section from the docs for now. What should I do with the `kustomize` and `Makefile`? Is it ok to leave the GCR there?

Reply:
Since this is in alpha for now and anybody can build the image from the source for local use going with GCR seems ok for us for now. When the operator is close to release and integrated into CI, we shall push images publicly to dockerhub using CI.

Reply:
right, for k8s, we can just push to dockerhub and make it public. No one is gna run this in production until we support it, so should be fine.

Reply:
I will push this change to the next PR.